### DISCUSSION TITLE
module_save_time_logger : not logging in log4j-20240718-0917-12.log file

### DISCUSSION THREAD

**Discussion by ukrishne** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.interceptor-manager.hook-samples#15] [Q&A] [answered]
Dear Team,

I have downloaded latest code and deployed jars as shown in the image. am trying to check module_save_time_logger is logigng for saving module. but i cant find entry in the log file. (polarion has been restarted too)
![image](https://github.com/user-attachments/assets/2d081e6a-a36a-4008-b11b-eac4c886ed5f)

i am not sure if the below message is coming from module_save_time_logge hook or not 
![image](https://github.com/user-attachments/assets/e7653e6f-cff1-44b5-9bd2-4c90a6c96a3b).

Please clarify.

Thanks in Advance
Umesha Krishnegowda

**Comment by grigoriev** [comment]
Hi @ukrishne,

by default all new hooks are disabled. You should enable required hooks in Admin pane.

Go to `http://<your host>/polarion/#/administration/Interceptor/Settings` and enable corresponding hook:
<img width="348" alt="image" src="https://github.com/user-attachments/assets/9cdae057-827d-496c-970f-1588762fe82c">

**Reply by grigoriev** [reply]
Looks like you don't have Interceptor extension installed in your Polarion. Am I right?

**Reply by ukrishne** [reply]
yes. i dont have Interceptor extension installed in your Polarion. i am not able to see the interceptor extension as shown below
![Uploading image.png…]()

**Reply by ukrishne** [reply]
should i need to install https://extensions.polarion.com/extensions/146-work-item-action-interceptor-framework  ??

**Reply by grigoriev** [reply]
No, we have developed Interceptor extension. This extension is not public now. As soon as it goes open source you will get access.

**Reply by ukrishne** [reply]
ok thanks

**Reply by grigoriev** [reply, marked as answer]
Hi @ukrishne,

now the [interceptor-manager](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.interceptor-manager/) is public.

**Reply by grigoriev** [reply]
please use the latest versions of interceptor-manager and hook samples

**Reply by ukrishne** [reply]
yes. i did that and it worked. many thanks for making it open source.
